databricks/koalas/tests/test_expanding.py

```diff
@@ -115,7 +120,10 @@ def _test_groupby_expanding_func(self, f):
             repr(getattr(pdf.groupby([("a", "x"), ("a", "y")]).expanding(2), f)().sort_index()))
 
     def test_groupby_expanding_count(self):
-        self._test_groupby_expanding_func("count")
+        # The behaviour of ExpandingGroupby.count are different between pandas>=1.0.0 and lower,
```

Comment:
@itholic, do we handle all of TODOs here? I still see many TODOs left, for example, in https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-b2490bcc905f49fa2f41e71c72121af8R780

- If and only if the behaviour in pandas 1.0.0 is a bug, we skip or test differently. We should explain in the comments why it looks a bug. Ideally, we should link the pandas issue where the behaviour was changed.
- Otherwise, we match to latest pandas' behaviour.

Reply:
@HyukjinKwon 

Thanks for the comment !

> do we handle all of TODOs here?

I think all TODO has been resolved yesterday before the #1197 merged, except two.

- one is: https://github.com/databricks/koalas/pull/1197/files#r381830438 
  - since it seems requiring lots of effort, i'd like to work this in separated PR.
- and the other is: https://github.com/databricks/koalas/pull/1197/files#r382345421 which i made here.

am i missing something?

Reply:
Let's do it all here in this PR. Does this PR handle all of them below?

https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-d3f67c2eab2a8dd3df7283426995819fR86
https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-497e17c16cdf5dca7bbc65d1b130574eR127
https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-497e17c16cdf5dca7bbc65d1b130574eR750
https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-b2490bcc905f49fa2f41e71c72121af8R780
https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-bbc5f26000bddf5eb70586e69d18f8f8R57

We should describe why it looks like a bug in pandas, ideally with some issue or PR links in pandas side.

Reply:
Ah, i thought you mean the comment in the previous PR you left yesterday.

okay i just changed the title of this PR to `Matching the behaviour to pandas>=1.0.0 at most.`, and will start cleaning up remaining TODOs right away.

Thanks! :)

Reply:
- [ ] [24726bd#diff-d3f67c2eab2a8dd3df7283426995819fR86](https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-d3f67c2eab2a8dd3df7283426995819fR86)
- [ ] [24726bd#diff-497e17c16cdf5dca7bbc65d1b130574eR127](https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-497e17c16cdf5dca7bbc65d1b130574eR127)
- [ ] [24726bd#diff-497e17c16cdf5dca7bbc65d1b130574eR750](https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-497e17c16cdf5dca7bbc65d1b130574eR750)
- [ ] [24726bd#diff-b2490bcc905f49fa2f41e71c72121af8R780](https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-b2490bcc905f49fa2f41e71c72121af8R780)
- [ ] [24726bd#diff-bbc5f26000bddf5eb70586e69d18f8f8R57](https://github.com/databricks/koalas/commit/24726bdf3c69f224c77152361b9641ce6a2408df#diff-bbc5f26000bddf5eb70586e69d18f8f8R57)